### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by aladdin91**
this is my home work for js week1

### CODE REVIEW

File: javascript/javascript1/week1/smart-ease io/Ez-Namey.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+firstWord = ['WestCoast', 'Smooth', 'Network', 'servo', 'New', 'Heavenly', 'Giant', 'Brighter', 'Pioneer', 'AllPro']
```
Comment (naresh699) [inline javascript/javascript1/week1/smart-ease io/Ez-Namey.js:1]: any specific reason to keep 'firstWod' and 'secondWords' without any declaration key 'like var, let or const'

File: javascript/javascript1/week1/smart-ease io/Housey-pricey.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    console.log(`Peter paid too much`)
+}
+
+const JuliaHouseWidth = 5
+const JuliaHouseDepth = 11
+const JuliaHouseHeight = 8
+const JuliaGardenSizeInM2 = 70
+const JuliaHouseCost = 1000000
+const JuliaVolumeInMeters = JuliaHouseWidth * JuliaHouseDepth * JuliaHouseHeight
+JuliaHousePrice = JuliaVolumeInMeters * 2.5 * 1000 + JuliaGardenSizeInM2 * 300;
```
Comment (naresh699) [inline javascript/javascript1/week1/smart-ease io/Housey-pricey.js:21]: declare variable with 'var/let/const' etc. Please check google , how it is different from if we declare without keys'

**Review by naresh699 — COMMENTED** [review]
good work Alaa.
"use strict"; on the top of your JS. check the error shown on the page.
Also read about "use strict' or keeping It loose validation (without)

File: javascript/javascript1/week1/smart-ease io/Ez-Namey.js (hunk 1 above)
Comment (aladdin91) [reply javascript/javascript1/week1/smart-ease io/Ez-Namey.js]: oh no, actually i forgot the declaration

**Pull request merged**